src/lib/y2partitioner/widgets/overview.rb

```diff
@@ -0,0 +1,246 @@
+require "cwm/widget"
+
+require "y2partitioner/icons"
+
+Yast.import "Hostname"
+
+module Y2Partitioner
+  module Widgets
+    # Widget representing partitioner overview.
+    #
+    # It has replace point where it displays more details about selected element
+    # in partitioning.
+    #
+    # TODO: abstract treewidget from it
+    class Overview < CWM::CustomWidget
+      # creates new widget for given device graph
```

Comment:
What about using yardoc to document params? The same applies to other methods like `#handle`.